README.md

```diff
@@ -1,7 +1,8 @@
 Testify - Thou Shalt Write Tests
 ================================
 
-ℹ️ We are working on testify v2 and would love to hear what you'd like to see in it, have your say here: https://cutt.ly/testify
+Notice about Testify V2:
+It won't happen due to various reasons. You can read more about it on the maintainer's manifesto [here](https://github.com/stretchr/testify/issues/1089#issuecomment-1812734472).
```

Comment:
```suggestion
> [!NOTE]
> Testify is being maintained at v1, no breaking changes will be accepted in this repo. [See discussion about v2](https://github.com/stretchr/testify/discussions/1560).
```

Suggest using a proper GitHub markdown note and remove the emotional language from the note.

Reply:
Hi @brackendawson 
Thank you for finally coming to a concensus, I have applied your suggestion. 😄